pkg/github/github.go

```diff
@@ -147,13 +158,13 @@ func (c *Client) getLabels(ctx context.Context, owner, repo string) ([]Label, er
 	return labels, nil
 }
 
-func (c *Client) updateLabel(ctx context.Context, owner, repo string, label Label) error {
+func (c *Client) updateLabel(ctx context.Context, owner, repo, labelName string, label Label) error {
 	l := &github.Label{
 		Name:        &label.Name,
 		Description: &label.Description,
 		Color:       &label.Color,
 	}
-	_, _, err := c.githubClient.Issues.EditLabel(ctx, owner, repo, label.Name, l)
+	_, _, err := c.githubClient.Issues.EditLabel(ctx, owner, repo, labelName, l)
 	fmt.Printf("label %+v updated on: %s/%s\n", label, owner, repo)
```

Comment:
Looks like you are no longer passing the label, but the labelName. But label is still used in this print statement.